Source/engine/actor_position.hpp

```diff
@@ -5,21 +5,21 @@
 namespace devilution {
 
 struct ActorPosition {
-	Point tile;
+	PointOf<uint8_t> tile;

```

Comment:
`using WorldTile = PointOf<uint8_t>` would be useful

Reply:
Done